firmware/application/apps/analog_audio_app.hpp

```diff
@@ -133,6 +133,23 @@ class SPECOptionsView : public View {
         1,
         ' ',
     };
+    Text text_rx_cal{
+        {19 * 8, 0 * 16, 11 * 8, 1 * 16},  // 18 (x col.) x char_size,  12 (length) x 8 blanking space to delete previous chars.
+        "Rx_IQ_CAL  "};
+    NumberField field_rx_iq_phase_cal_2837{
```

Comment:
I think it would save a few bytes if there was only a single NumberField, instead of two.  You can use the "set_range" method to modify the range based on whether it's hackrf_r9.  (I can do this optimization in a subsequent PR if you like.)

Reply:
Thanks Mark for your revision , advice and approval . 
As I am not familiar with this "set_range",  I would  appreciate a lot ,  if you can optimize it with next subsequent  PR , And in that next PR, please also do the same in the other Tx_iq_phase calibration,   "firmware/application/apps/ui_mictx.hpp" ==> I also used there same double  NumberField structure to handle 5 / 6 bits range in TX side feature  .   thanks,  then let's merge it  as it is now, and I will learn about it in your next PR .